__tests__/api.test.js

```diff
@@ -40,4 +40,25 @@ describe("News API BACKEND PROJECT", () => {
             })
         })
     })
+    describe("GET /api that returns documentation of available api endpoints", () => {
+
+        test('returns a 200 response status', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+        })
+        test('returns a documentation with the expected properties', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(typeof body).toBe("object")
+                const objectKeys = Object.keys(body)
+                expect(objectKeys.includes("GET /api")).toBe(true)
+                expect(objectKeys.includes("GET /api/topics")).toBe(true)
+                expect(objectKeys.includes("GET /api/articles")).toBe(true)
+            })
```

Comment:
This is a nice start, but there is an even easier and more thorough way of doing this. 

The endpoints file is a .json file which means it can be required in (exactly like your controller) so this test simply needs to assert that what we are receiving from `body` is the same object contained in the endpoints.json file, and thats about as complex as this really needs to be.